build/ios/scripts/build_arch.sh

```diff
@@ -56,15 +49,15 @@ export pkg_make_flags=-j4
     test -x ./configure || ./autogen.sh
     ./configure -q --disable-shared \
```

Comment:
You already pass `--disable-shared` as part of `$EXTRA_CONFIG` below. Thus, I'd remove it from here.